File: generator-apeer/generators/app/templates/matlab/src/apeer_main.m
function [] = apeer_main(varargin)

    adk = ApeerDevKit(varargin{:});
    inputs = adk.get_inputs();

    outputs = your_code(<%- module_inputs_matlab_adk %>);

    <%- module_outputs_matlab_adk %>

    adk.finalize();

end

File: generator-apeer/generators/app/templates/python/apeer_main.py
from apeer_dev_kit import adk
import your_code

if __name__ == '__main__':
    inputs = adk.get_inputs()

    outputs = your_code.run(<%- module_inputs_python_adk %>)

    <%- module_outputs_python_adk %>

    adk.finalize()
